encrypted-media/content/drmconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+    "com.widevine.alpha": {
+        "drmtoday": true,
```

Comment:
What does this control? Is it necessary or do we just need a server URL? Is it intended to mean "not Clear Key?" I think we should avoid assuming any specific license server provider in case someone wants to add another later.